.github/scripts/removeIssuesFromDesignProjects.js

```diff
@@ -52,7 +52,7 @@ try {
     const deleteQuery = `mutation { deleteProjectV2Item(input: {projectId: "${projectItem.project.id}", itemId: "${projectItem.id}"}) { clientMutationId } }`;
     runQuery(deleteQuery);
     createComment(
-      `The issue has been removed from the [${projectItem.project.title}](${projectItem.project.url}) project.`,
+      `This issue has been removed from the [${projectItem.project.title}](${projectItem.project.url}) project.`,
```

Comment:
Nitpick: since this is also editing the comment, perhaps the PR title could be renamed to reflect both changes? E.g., `refine issue-from-design-projects workflow` or something similar.